src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
```suggestion
    background-position: center;
    
    &:nth-child(...) {
      background-image: url("../images/star-active.svg");
    }
  }
```

Reply:
you can combine that in one selector

Reply:
Thanks, good decision, but unfortunately I can not use that because scss push me some error and I don't know how to fix it!